src/gui/sdl_mapper.cpp

```diff
@@ -616,7 +615,14 @@ class CStickBindGroup : public CBindGroup {
 		emulated_hats=0;
 		JOYSTICK_Enable(emustick,true);
 
+		// NOTE: the _stick number passed in here is *one* index SDL uses to
+		// find the joystick, but it's not the one that will appear later in events.
+		// (see https://wiki.libsdl.org/SDL2/SDL_JoystickInstanceID)
```

Comment:
Nice comments but can you be clearer in your wording and use SDL terminology consistently? Let's not add our terms as well to the mix 😄 

We should only talk about "joystick index (Nth joystick in the system" and "SDL joystick instance ID".

Reply:
Also, I suggest refactoring `_stick` to `_stick_index`.

Reply:
Honestly, I think it would be best to simply quote the SDL doco here. I find their wording clearer as you're still talking about "two different indexes", where in reality there is no such thing: there is the _one_ index, the device index, which is used for opening the device, then after that there's the instance ID in SDL events. That's all there is to it.

This device index as seen by the OS and instance ID in events distinction is very clear; talking about two different indexes muddies the waters and makes this appear more complicated than it really is.

So I suggest to just replace your comment with this:

```
From the SDL doco (https://wiki.libsdl.org/SDL2/SDL_JoystickOpen): 

  The device_index argument refers to the N'th joystick presently recognized
  by SDL on the system. It is NOT the same as the instance ID used to identify
  the joystick in future events.

Also see: https://wiki.libsdl.org/SDL2/SDL_JoystickInstanceID
```

Reply:
Done.